citizen-science-theory-and-practice.csl

```diff
@@ -0,0 +1,267 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Citizen Science: Theory and Practice</title>
+    <id>http://www.zotero.org/styles/citizen-science-theory-and-practice</id>
+    <link href="http://www.zotero.org/styles/citizen-science-theory-and-practice" rel="self"/>
+    <link href="http://www.zotero.org/styles/sociology-of-health-and-illness" rel="template"/>
+    <link href="http://theoryandpractice.citizenscienceassociation.org/about/submissions/#References" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="sociology"/>
+    <category field="social_science"/>
+    <category field="anthropology"/>
+    <category field="medicine"/>
```

Comment:
@damnation333, based on http://theoryandpractice.citizenscienceassociation.org/about/, maybe we should just limit this to the "science" category?